hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -465,9 +495,14 @@ protected void finalize() throws Throwable {
     super.finalize();
     if (!closed) {
       close(); // releasing resources is more important than printing the warning
-      String trace = Joiner.on("\n\t").join(
+      String trace = Joiner.on("\\n\t").join(
           Arrays.copyOfRange(createStack, 1, createStack.length));
-      LOG.warn("Unclosed input stream created by:\n\t{}", trace);
+      LOG.warn("Unclosed input stream created by:\\n\t{}", trace);
     }
   }
+
+  @Override
+  public String toString() {
+    return "HiveCatalog [name=" + name + ", location=" + this.conf.get("hive.metastore.uris") + "]";
```

Comment:
In configuration, we use `uri` instead of `location`. Can you use that here?

Also, other `toString` methods use the helpers in `MoreObjects`. To have a common format, could you use that instead?

Reply:
Done